src/AccessControl/tests/testModuleSecurity.py

```diff
@@ -67,6 +67,9 @@ def testPublicModule(self):
         self.assertAuth('AccessControl.tests.public_module.submodule',
                         ('pub',))
 
+    def test_star_import_not_allowed(self):
+        self.assertUnauth('AccessControl.tests.public_module', ('*',))
+
```

Comment:
What is the difference to the next test (`test_public_module_asterisk_not_allowed`)?

Reply:
That was a left-over from some experimentation, I've backed it out now.